helmfile.d/upstream/index.yaml

```diff
@@ -53,6 +54,7 @@ charts:
 
   kubereboot/kured: 4.5.1
 
+  kubernetes-external-dns/external-dns: 1.14.3
```

Comment:
Could you bump this to 1.14.4? It includes the CRD now.

Reply:
Missed the 1.14.4 chart as it is not part of a release :+1: